recipes/vlfeat/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.9.20" %}
+
+package:
+  name: vlfeat
+  version: {{ version }}
+
+source:
+  fn: vlfeat-{{ version }}-bin.tar.gz
+  url: http://www.vlfeat.org/download/vlfeat-{{ version }}-bin.tar.gz
+  
+  patches:
+    - no_snprintf.patch  # [win and py35]
+    - no_crt_copy.patch  # [win]
+
+build:
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}                # [win]
```

Comment:
As an example of this SCons point mentioned above, consider building MongoDB ( https://github.com/conda-forge/staged-recipes/pull/392 ). This builds with SCons, but will try to use Boost if available. If Boost is not available, it tries to build some subset of Boost. However, we have a dilemma. Do we spend time rebuilding something we have (Boost)? Or do we artificially restrict MongoDB to Python 2 in some way even though it does not depend on Python at all.

At least for my current problem, I'm seriously debating doing the same thing Homebrew did, which is to break Boost into a non-Python part and the Boost.Python part ( https://github.com/conda-forge/boost-feedstock/issues/1 ). That way if something needs Boost, but doesn't care about Python it doesn't suffer some weird restriction on where it can be installed.